packages/camera/camera_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.4.0
```

Comment:
Why is this a breaking change @maciejbrzezinski ?

Looking at https://github.com/flutter/plugins/pull/6533, if we launch this, users will have 2 authorization popups, but the feature would still continue working, right?

Reply:
I assume it as a breaking change, because there may be scenario when someone was using this method to request audio permission, now he will have to use something else, so the API of the plugin has been changed.

Reply:
Ah, I think the moment they request a camera+audio after retrieving the available cameras, the system will prompt the user for audio permission (if needed!). Let's go with this one!